src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -330,7 +358,9 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
             if fovs_processed % 10 == 0 or fovs_processed == len(fovs_list):
                 print("Processed %d fovs" % fovs_processed)
 
-    # get mean 99.9% across all fovs for all markers
+        # get mean 99.9% across all fovs for all markers, check that none are missing
+    quant_dat = pd.read_csv(quantile_path, index_col="channel")
+
     mean_quant = pd.DataFrame(quant_dat.mean(axis=1))
```

Comment:
It's a tossup for me whether or not we should delete `quantile_data.csv` after this function is done, personally I would but up to you.

Reply:
I think we should delete this file as well. I think having this file stick around would just confuse people.

Reply:
It only is there until `pixel_som_clustering.generate_som_avg_files()` is ran 2 cells later in the notebook, then it's deleted. Not quite sure why? Is the pixel_mat_data subdir emptied and the feather files regenerated?? @alex-l-kong @cliu72

Reply:
Hmm I think I'm a little confused - it seems like you're creating a file called `quantile_data.csv` in `data_dir` to store the quantiles for each FOV right? I think we meant that at the end of the `create_pixel_matrix` function after the mean quantiles has been calculated and stored in the feather file, we can delete `quantile_data.csv` since it's no longer needed. Should be independent of the `cluster_pixels` function.

Reply:
Oh okay, I can delete it at the end of the function! I was just curious why it gets deleted on it's own later in the notebook anyways.

Reply:
Ohh I see what you mean. Yeah so the reason it's implemented like that is to deal with when someone restarts the notebook during cluster assignment. So like all the FOV feather files are created in `pixel_mat_dir` during preprocessing, and then during cluster assignment, the feather files that have the cluster assignments are written to `pixel_mat_dir_temp` (because if they were written to `pixel_mat_dir` directly, then it would overwrite the originals we wouldn't be able to tell which FOVs were already clustered without iterating through all the FOVs to check and that would take longer). So we write to the temp directory and then after all the clustering is done, then we delete the original `pixel_mat_dir` and replace it with the temp one. Hopefully that made sense.

I think we can delete quantile_data separately from all that though, just to keep it a bit cleaner.